third_party/blink/brave_page_graph/page_graph.cc

```diff
@@ -143,6 +147,10 @@ PageGraph::PageGraph(Document& document) :
     cookie_jar_node_(new NodeStorageCookieJar(this)),
     local_storage_node_(new NodeStorageLocalStorage(this)),
     session_storage_node_(new NodeStorageSessionStorage(this)),
+    canvas_node_(new NodeWebAPI(this, "Canvas")),
```

Comment:
I think there is some small reworking that needs to be done here.  "Canvas" isn't a WebAPI, but "Canvas.prototype.toBlobURL" is.  So, i'd expect a single NodeWebAPI node for each method you instrument. It makes some kinds of analysis much easier (constant time answers for "how many scripts called "toBlobURL", etc).

Could you rework as above?